packages/rds/src/components/Image/Image.module.scss

```diff
@@ -0,0 +1,12 @@
+@layer rds.component {
+  .picture {
```

Comment:
I've had a lot of success using the old "FifeImage" component but without any defined styles (letting the caller specify styles, and having the emitted `<img>` behave like the default `<img>` without any styling from the core component. Do you think we need the `Image` component to come with styles?

Reply:
The styles here are pretty minimal and would be the same reset style I generally use for `img` tags. I can remove these styles here in favor of having a base stylesheet that normalizes our `img` (and `svg`, `video`) styles.

Reply:
Yeah; I think it'd be better to have the `<Image>` component not apply styles (at least by default). For example, if I wanted to have an inline image appear without taking up the full width of the container I'd need to overwrite these styles.

Reply:
Counter-point: in more cases than not, I'd expect the image to want to take the full-width of its container, and only in cases where I don't want it to would I override the style (to either `width: auto` or `width: 50%` or `max-width: 500px`). I usually think if a user uploads an image smaller than the container and it doesn't fill the whole width, it'd be considered a bug.

But again, I'm happy to defer that to a base stylesheet.